docs/PythonAPIOverview.md

```diff
@@ -66,25 +66,26 @@ from onnx import AttributeProto, TensorProto, GraphProto
 
 # Create one input (ValueInfoProto)
 X = helper.make_tensor_value_info('X', TensorProto.FLOAT, [1, 2])
+Z = helper.make_tensor_value_info('Z', TensorProto.FLOAT, [1, 2])
```

Comment:
Thank you very much for improving the tutorials!

Do you mind to add comment in this doc to clarify the operator set versioning please? say, this is based on Pad-2 or maybe Pad-5.

Reply:
otherwise, the same confusion will happen when Pad updated in future again.

Reply:
Ok. I've done the necessary changes and created a new pull request that seems to have added to this one. Hope everything is ok.

Thank you for the input!